src/components/input/bl-input.css

```diff
@@ -44,6 +45,10 @@
   --icon-size: var(--bl-font-body-text-3-line-height);
 }
 
+.wrapper:has(input:-webkit-autofill, input:-webkit-autofill:hover, input:-webkit-autofill:focus) {
```

Comment:
Because of [browser support for `:has` selector](https://caniuse.com/?search=%3Ahas) not perfect yet, I see this will not make the experience of some users about auto filling very good. But I also see that, trying to cover older browsers will need some work on JS and this will add performance costs for every user. So I think it's ok to keep this in that way for now.

My comments below about vendor prefix and hover/focus selectors also applies here.